docs/misc/docs/mygui-building.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Building MyGUI
+---
+
+# Building MyGUI
+
+Building MyGUI was kinda painful, so it's time to document the entire process so it's not lost to time.
+
+# Getting MyGUI
```

Comment:
For Markdown you should structure your headings as a hierarchy, so # is the title and ## a subsection and ### a subsection.